### ISSUE TITLE
Merging if conditions looses inner statement

### ISSUE THREAD

**Issue by cal101** [walkmod/walkmod-pmd-plugin#2]
```
     protected AdminManagingCommand getAdminManaging() {
         AdminManagingCommand ret = (AdminManagingCommand) (ctx.getCommandController().getCommand(AdminManagingCommand.NAME));
-        if (ret == null) {
-            if (log.isDebugEnabled()) {
-                log.debug("Command AdminManagingCommand was not loaded. If you need to debug the AdminServer please load AdminManagingCommand in AdminConfig.prop.");
-            }
+        if ((ret == null) && log.isDebugEnabled()) {
         }
         return ret;
```

**Comment by rpau** [comment]
Ok, I will create a test for it.

**Comment by rpau** [comment]
Same than issue #3 
@cal101 , just to be sure. Have you replaced javalang in the .ivy folder instead of using the last one from maven repository?

**Comment by cal101** [comment]
Yes, I think so because I am doing it via a shell script.
But let me try to make real test cases in the evening.

**Comment by rpau** [comment]
Cal, you were right.
I have fixed it, in this commit : 4d2c291c0d9eba4c1791ae6d81e084545a17498e
